### PR TITLE
count optimization

### PR DESCRIPTION
**Pull request by samsonasik**
- [x] Is this related to quality assurance?

### CODE REVIEW

File: src/Element/Collection.php [hunk 1]
```diff
@@ -172,7 +172,8 @@ public function setObject($object)
         }
 
         $this->object = $object;
-        $this->count  = count($object) > $this->count ? count($object) : $this->count;
+        $countObject  = count($object);
+        $this->count  = $countObject > $this->count ? $countObject : $this->count;
```
Comment (webimpress) [inline src/Element/Collection.php]: We can use `max`, and I think it is micro better:
`max`: https://3v4l.org/3neNN/vld#output
`?:`: https://3v4l.org/5tWTS/vld#output

File: src/View/Helper/FormRow.php [hunk 2]
```diff
@@ -141,7 +141,7 @@ public function render(ElementInterface $element, $labelPosition = null)
         }
 
         // Does this element have errors ?
-        if (count($element->getMessages()) > 0 && ! empty($inputErrorClass)) {
+        if ($element->getMessages() && ! empty($inputErrorClass)) {
```
Comment (webimpress) [inline src/View/Helper/FormRow.php]: Do we need `! empty` here or we can just have `!` ?

File: src/Element/Collection.php (hunk 1 above)
Comment (samsonasik) [reply src/Element/Collection.php]: done ;)

File: src/View/Helper/FormRow.php (hunk 2 above)
Comment (samsonasik) [reply src/View/Helper/FormRow.php]: done ;)

File: src/Element/Collection.php [hunk 3]
```diff
@@ -172,7 +172,8 @@ public function setObject($object)
         }
 
         $this->object = $object;
-        $this->count  = count($object) > $this->count ? count($object) : $this->count;
+        $countObject  = count($object);
```
Comment (webimpress) [inline src/Element/Collection.php]: It is used now, so we don't need it, imho.

Comment (samsonasik) [reply src/Element/Collection.php]: done ;)

**Comment by weierophinney** [comment]
Thanks, @samsonasik.

**Pull request merged**